src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Consider giving this property a more descriptive name since `by` can be understood as either the person who created the task or the deadline of the task